dyn_em/module_initialize_real.F

```diff
@@ -3089,6 +3104,16 @@ SUBROUTINE init_domain_rk ( grid &
 
       !  Split NUDAPT  Urban Parameters
 
+      distributed_aerodynamics_if: IF (config_flags%use_distributed_aerodynamics) THEN
+         DO j = jts , MIN(jde-1,jte)
+            DO i = its , MIN(ide-1,ite)
+              IF (grid%ivgtyp(i, j) == model_config_rec%isurban(grid%id)) THEN
```

Comment:
need to also include LCZ type in the if-statement for urban pixels to allow working with LCZ capability.

Reply:
@cenlinhe  This is fixed.

Reply:
@cenlinhe I added if-statement and error messages to `share/module_check_a_mundo.F`.